00_Base/src/services/credentials.service.ts

```diff
@@ -680,43 +775,13 @@ export class CredentialsService {
       serverVersionUrl,
     );
     const postCredentials: CredentialsDTO = postCredentialsResponse.data;
-    const credentialsTokenC = postCredentials.token;
-    clientInformation.clientToken = credentialsTokenC;
-
-    const clientCredentialsRoles = postCredentials.roles;
-    // remove old roles that were passed in via endpoint // todo maybe dont set roles in endpoint then?
-    for (const clientCredentialsRole of clientInformation.clientCredentialsRoles) {
-      // todo can this be done in one query??
-      await clientInformation.$remove(
-        ClientInformationProps.clientCredentialsRoles,
-        clientCredentialsRole,
-      );
-      await clientCredentialsRole.destroy();
-    }
-    // add new
-    const newClientCredentialsRoles = clientCredentialsRoles.map(
-      (credentialsRoleDTO) =>
-        ClientCredentialsRole.create(
-          {
-            ...(credentialsRoleDTO as Partial<ClientCredentialsRole>),
-            [ClientCredentialsRoleProps.clientInformationId]:
-              clientInformation.id,
-          },
-          {
-            include: [
-              {
-                model: BusinessDetails,
-                include: [Image],
-              },
-            ],
-          },
-        ),
-    );
-    clientInformation.setDataValue(
-      ClientInformationProps.clientCredentialsRoles,
-      newClientCredentialsRoles,
+
+    // TODO we remove old client roles that were passed in via endpoint
+    //  maybe dont set roles in endpoint then?
```

Comment:
This is the comment in the red line 687. Since I reuse this piece of code but I don't want to lose this comment, I move it to here. Sorry for the confusion. I should make the comment more clear after moving it.

Before my refactor, this piece of code works for the registration API. `endpoint` should refer to registration API. In that api, we at first create some client roles and then after some actions, we delete existing ones and insert new ones (red line 690 and 699). I think this comment is to suggest not to set those client roles initially at all but only store the updated client roles. @elliot-sabitov please let me know if my understanding is correct.

Reply:
I think these comments are no longer relevant and can be removed